src/controllers/authController.js

```diff
@@ -55,8 +112,25 @@ const login = async (req, res) => {
 };
 
 //logout endpoint
-
 const logout = async (req, res) => {
+  /*
+    #swagger.summary = 'Logout a user'
+    #swagger.parameters['token'] = {
+        in: 'cookie',
+        description: 'JWT token in the cookie',
+        required: true,
+        type: 'string'
+    }
```

Comment:
oh interesting I didn't realize we could define the cookie value here. Were you able to get it to work with the "Try It" feature? Like did it actually send the cookie entered ? 

I think otherwise, we can just leave this out, and the browser can handle setting/fetching the cookie as it usually would.

Reply:
I'll look into a bit more today to understand better how it works with swagger-autogen :thinking:

Reply:
I read the Swagger documentation, and it doesn't explicitly specify how to document the 'token' parameter, which in my code is expected to be a JWT token in a cookie. I included the 'token' parameter with the description 'JWT token in the cookie' in the code comments because it might be useful information for other developers. I understand that my approach might not be entirely accurate😅, and I couldn't test it due to the errors in the code above.

Reply:
What you have works and does follow the Swagger docs :white\_check\_mark:  My recommendation to change it is mainly just to make the API docs easier to navigate for everyone. 

The browser already takes care of storing and passing along values in cookies. But if we add your line here, then the user using the api docs has to manually grab the token value and paste it into the box to try out any authenticated enpoint, which feels like extra friction that we don't really need.

Reply:
Thank you very much, so helpful for me ❤️

Reply:
<img width="686" alt="swagger1" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/cd563b40-6cc9-4d5e-85e3-f776c0ac5677">